CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 
 ### Changes
 
+* [#4157](https://github.com/bbatsov/rubocop/issues/4553): Add `node_modules` to default excludes. ([@iainbeeston][])
```

Comment:
Thanks @Drenmi - that's really helpful of you! (and my bad - copy-paste... 😮 ) Should be fixed now